providers/base/units/stress/boot.pxu

```diff
@@ -72,9 +71,7 @@ depends: init-boot-loop-data
 id: cold-boot-loop-reboot{reboot_id}
 category_id: stress-tests/cold-boot
 _summary: Perform cold reboot {reboot_id}
-_description: This is a template that will be used to generate a stress test
-  of the system boot. Specifically this is how the device will be request a
-  reboot.
+_description: Enter sleep mode after 240s.
```

Comment:
Also configurable with the default being 120s. It should wake up after another 120.